easybuild/easyblocks/c/cblas.py

```diff
@@ -56,8 +56,11 @@ def configure_step(self):
         self.cfg.update('buildopts', 'FC="%s"' % os.getenv('F77'))
         self.cfg.update('buildopts', 'CFLAGS="%s -DADD_"' % os.getenv('CFLAGS'))
         self.cfg.update('buildopts', 'FFLAGS="%s -DADD_"' % os.getenv('FFLAGS'))
+
+        extra_blas_libs = '-lmkl_intel_thread -lmkl_lapack95_lp64 -lmkl_intel_lp64 -lmkl_core'
```

Comment:
@Darkless012 This should be done conditionally, only when installing `CBLAS` with a toolchain that includes `imkl`?

Reply:
oki, will fix that. Thanks :)